studies/templates/studies/study_edit.html

```diff
@@ -118,7 +150,7 @@ <h1 id="manage-researchers" class="panel-title">Manage Researchers</h1>
                 {% if messages %}
                     {% for message in messages %}
                         {% if message.extra_tags == 'user_added' or message.extra_tags == 'user_removed' %}
-                            <div id="add-researcher-messages" class="alert alert-success alert-dismissable">
+                            <div id="add-researcher-messages" class="alert {{ message.tags }} alert-dismissable">
```

Comment:
I'm normally not looping through messages, most of the site just does {% bootstrap_messages %} but I wanted more control over where certain success messages show on this page.  The researcher messages I needed to show directly over the researchers.  Also, when researchers are edited, I am using x-editable, so I had to build the success messages myself since the page isn't reloading.